Content/LuaScripts/ExampleViewModel.lua

```
@@ -0,0 +1,213 @@
+    elseif propertyName == "healthDisplay" then
+        return viewModel.playerHealth .. " / " .. viewModel.playerMaxHealth
+    elseif propertyName == "manaDisplay" then
+        return viewModel.playerMana .. " / " .. viewModel.playerMaxMana
+    end
+    
+    -- Return nil to use default behavior
+    return nil
+end
+
+-- Called when a property is set (optional)
+function OnSetProperty(viewModel, propertyName, value)
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'OnSetProperty'

(W111)